aptos-utils/src/lib.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright © Aptos Foundation
+
+/// An internal implementation to imitate the feature of `try` in unstable Rust.
+/// Useful to use '?' chaining on option/result without the need to wrap the expression in a
+/// function.
+#[macro_export]
```

Comment:
add a comment stating that this macro is obsolete once:
https://github.com/rust-lang/rust/issues/31436
is resolved/try is stable.

Reply:
Done